main.go

```diff
@@ -68,7 +66,6 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(selfnoderemediationv1alpha1.AddToScheme(scheme))
-	utilruntime.Must(machinev1beta1.AddToScheme(scheme))
```

Comment:
and here the comment 😉 
I'm pretty sure we need this, I expect SNR to fail without it when we get a CR created by MHC, where we need to get the Machine

Reply:
Good point,
Since we intend to have MHC functionality moved to NHC I wonder if we can also leap into the conclusion that that eventually SNR will work only  with NHC ? :thinking: 
If so we can put this PR "on ice" until then.

Or maybe there is a better option that I'm missing , WDYT ?

Reply:
> until then

"then" won't be in the near future... why not just keeping this line here? 🤷🏼‍♂️

Reply:
IIUC the meaning of keeping this line is keeping the dependency in `machine-api-operator` which brings us back to the [initial issue](https://github.com/medik8s/self-node-remediation/pull/131#issuecomment-1616603651).

Reply:
yes, the machine API moved from `github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1` to `github.com/openshift/api/machine/v1beta1`, so we need to add it to our scheme from there. It's the same like in `selfnoderemediation_controller.go.`